### DISCUSSION TITLE
can't change namesilo name server to 'ns1.vercel-dns.com' or 'ns2.vercel-dns.com'

### DISCUSSION THREAD

**Discussion by liulei237136** [vercel/vercel#4414] [Help] [answered]
Hi,everybody.
  I bought  a domain in namesilo.com, but when I wanted to change its default name servers to "ns1.vercel-dns.com' and “ns2.vercel-dns.com” of vercel, namesilo.com show some errors as following:
 " There was a problem with the NameServers you entered for the following domains:
   xxxxxxx.com
   This is likely caused by the entered NameServers not being created yet. Please check the spelling of the NameServers you entered as well as checking to make sure they have been created at the applicable registry."
  so anybody know what's wrong with it ? I change the name servers of namesilo.com to other name servers such as "ns1.bluehost.com", it works and no errors.
  thanks.

**Comment by mcsdevv** [comment, marked as answer]
Hey @liulei237136, we did experience a brief issue with this which has now been resolved, I'm sorry for the inconvenience.

If you try this again, it will be successful.
